dataactvalidator/scripts/loaderUtils.py

```diff
@@ -13,11 +13,7 @@ def padFunction(self, field, padTo, keepNull):
         """Pads field to specified length."""
         if isnull(field) or not str(field).strip():
             if keepNull:
```

Comment:
Do we still need `keepNull` as a parameter? It looks like that option was removed above. Not sure if other places still rely on it.